app/policies/finance_profile_policy.rb

```diff
@@ -5,6 +5,10 @@ def index?
     !!user&.finance?
   end
 
+  def show?
+    !!user&.finance?
```

Comment:
Why `!!`? That turns nil (which is falsey) into false and anything not falsey into `true`. I've never seen this as useful in a flag.

Reply:
@plisovin :eyes: 
If I recall it was to cast `nil` to `false`